### ISSUE TITLE
RatingControl color fix

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#3925]
**Describe the bug**

With the new colors we introduced in the token-experiment branch, we broke RatingControl's unselected rest state - it's now almost invisible in both light and dark theme, and definitely doesn't meet any contrast ratio requirements. 

![image](https://user-images.githubusercontent.com/13142665/104255279-8a91bb00-542d-11eb-9dd4-5f3be99b8a1f.png)
![image](https://user-images.githubusercontent.com/13142665/104255286-92515f80-542d-11eb-8128-967d0d842872.png)

As a temporary solution, I would like to introduce a change in geometry and color. 
- For the geometry change, we can borrow from [the HC approach](https://github.com/microsoft/microsoft-ui-xaml/blob/3b48922c6d5dad09307122a7b6fa8203edb52f07/dev/RatingControl/RatingControl_themeresources.xaml#L40) and add `UnsetGlyph="&#xE734;"`.
- For the color change, let's update `RatingControlUnselectedForeground` to `TextFillColorSecondaryBrush`.

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/30848776/)

**Comment by mdtauk** [comment]
@YuliKl You may want to take a look at the InfoBar colours too... #3744

**Comment by chingucoding** [comment]
I would like to work on this.
